webpack.config.js

```diff
@@ -166,35 +167,52 @@ const webpackConfig = {
 	externals: [ 'electron' ]
 };
 
+const vendorLibs = [
+	'classnames',
```

Comment:
how'd you come up with this list? it's pretty different from what we have now.

Reply:
just seeing if i could make the dev speedup greater by putting more in it

Reply:
ah ok. I was hoping you'd worked something up that found all the deps under `node_modules` that `build` depends on and moved them into `vendor` :)